eng/pipelines/common/global-build-job.yml

```diff
@@ -54,12 +54,33 @@ jobs:
         ${{ if ne(parameters.isOfficialBuild, true) }}:
           value: ''
 
+      - ${{ if ne(variables['System.TeamProject'], 'public') }}:
+        - group: AzureDevOps-Artifact-Feeds-Pats
+
       - ${{ each variable in parameters.variables }}:
         - ${{ variable }}
 
     steps:
     - template: /eng/pipelines/common/clone-checkout-bundle-step.yml
 
+    - ${{ if ne(variables['System.TeamProject'], 'public') }}:
+      - ${{ if ne(parameters.osGroup, 'Windows_NT') }}:
+        - task: Bash@3
```

Comment:
Could we have done this with nuget authenticate plugin?

Reply:
I thought so at one point, but Matt recommended for this one (for cross org packages I believe).

Reply:
Ok, just curious as we already use NuGet authenticate for cross org restoring for the icb merge restore.

Reply:
@mmitche Any reason to prefer one over the other?